src/init/mod.rs

```diff
@@ -33,8 +63,8 @@ pub fn init_stub(shell_name: &str) -> io::Result<()> {
 
     let shell_basename = Path::new(shell_name).file_stem().and_then(OsStr::to_str);
 
-    let starship = path_to_starship()?.replace("\"", "\"'\"'\"");
-
+    let starship_native = path_to_starship_native()?.replace("\"", "\"'\"'\"");
+    let starship_posix = path_to_starship_posix()?.replace("\"", "\"'\"'\"");
```

Comment:
Keep those two together as `native` and `posix` named struct members :bulb:

Reply:
I did this, and also refactored the quote logic into method.